.github/workflows/release.yaml

```diff
@@ -7,11 +7,11 @@ name: "release"
 on:
   push:
     tags:
-      - '**'
-  # workflow_dispatch will let us manually trigger the workflow from GitHub actions dashboard.
-  # This is important because sometimes the macos build fails due to Docker installation.
-  # See https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/manually-running-a-workflow
-  workflow_dispatch:
+      - '*'
```

Comment:
simpler way to express all tags. we don't need workflow-dispatch on releases as we can force push the tag again in worst case.